cpp/src/components/utils.h

```diff
@@ -122,8 +122,8 @@ class Exception : public std::exception {
  */
 template <typename Type>
 void copy(Type *dst, const Type *src, size_t len, cudaStream_t stream) {
-    CUDA_CHECK(cudaMemcpyAsync(dst, src, len * sizeof(Type),
```

Comment:
It is important to check the return code of cudaMemcpyAsync. I noticed CUDA_CHECK has been commented. There are 3 occurrences in this file.

Reply:
Ok, done.